lib/client.js

```diff
@@ -94,7 +93,7 @@ export class Http3Client extends Http3WebTransport {
   /**
    * @param {ClientConnectedEvent} args
    */
-  onClientConnected (args) {
```

Comment:
Is it worth adding a linter for this?  [standard](https://www.npmjs.com/package/standard) or something?

Reply:
There should be already some linter config files in the project. So far I do not protect the branch with the linter, but I always check against it manually before committing.

Reply:
Might be worth adding an npm script for it and running it in CI.  Will submit a PR.

Reply:
Ok, good idea, had just not a high priority, while I was alone.

Reply:
Completely fair!